CONTRIBUTORS.md

```diff
@@ -8033,4 +8033,7 @@
 
 -[@stpeteriz](https://github.com/stpeteriz)
 
--[@prokys](https://github.com/prokys)
\ No newline at end of file
+-[@nursamyrzash](https://github.com/nursamyrzash)
+
+-[@prokys](https://github.com/prokys)
```

Comment:
Please insert your github name in the middle of the CONTRIBUTORS.md. not very bottom

Reply:
Make sure to do a git pull for latest change in the master branch before anything.